Examples/Examples/FeatureFormExampleView.swift

```diff
@@ -66,8 +66,6 @@ struct FeatureFormExampleView: View {
                     if let featureForm = model.featureForm {
                         FeatureFormView(featureForm: featureForm)
                             .validationErrors(validationErrorVisibility)
-                            .padding(.horizontal)
-                            .padding(.top, 16)
```

Comment:
Please also remove these lines from `FeatureFormViewStep13.swift` onward to keep the tutorial in sync with the example.